src/platform/platform_winuser.c

```diff
@@ -423,6 +423,16 @@ QuicRandom(
 
 #endif
 
+#ifdef DEBUG
+#ifdef _WIN64
+// 64 bit
+#define AllocOffset 16
+#else
+// 32 bit
+#define AllocOffset 8
+#endif
+#endif
```

Comment:
Can you just use `sizeof(size_t)`?

Reply:
Or a define of something else instead of hard coded?

Reply:
Done